jumpscale/packages/vdc_dashboard/frontend/components/marketplace/Solution.vue

```diff
@@ -58,7 +58,7 @@
               <template v-slot:item.actions="{ item }">
                 <v-tooltip top>
                   <template v-slot:activator="{ on, attrs }">
-                    <v-btn icon :href="`https://${item.Domain}/`" target="_blank">
+                    <v-btn icon :href="`https://${item.Domain}/`" target="_blank" :disabled="item.Domain === ''">
```

Comment:
should we add v-if on this button to not view it instead of making it disabled?
`<v-btn v-if="item.Domain" ...>`

Reply:
Resolved
![image](https://user-images.githubusercontent.com/11272864/106025318-4c54f680-60d1-11eb-994c-127c12e9eee2.png)